### DISCUSSION TITLE
Internal Server error 500 while uploading images in production

### DISCUSSION THREAD

**Discussion by khanharis87** [strapi/strapi#6514] [Legacy Topics]
**Describe the bug**
Getting Internal Server 500, when I try to upload in production. Works fine in development and production when I run local. MongoDB is connected, as there is no error when I build on server, I can do GET request on /upload/files, but on POST it throws 500. 
Also all the permissions are granted for public/authenticated user to upload in production.

**System**

- Node.js version: 12.16.7
- NPM version:6.14.4
- Strapi version:  3.0.0-beta.20 
- Database:MongoDb
- Operating system:Mac

**Additional context**
Changing permission in local did not give the desired affects. I can still upload, even after I have changed the permission to not allow.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
